config/module.config.php

```diff
@@ -17,12 +17,11 @@
  */
 
 use ZfrOAuth2\Server\AuthorizationServer;
```

Comment:
We are trying to go away from the ZF2 specific naming. So instead, here is what we've started doing on our internal apps using Middleware:

* `dependencies.global.php` : file that contains only dependencies key.
* `config.global.php`: this file provides some default basic configuration for the library (like an empty array for "grants" so that the app does not crash...).

Then, we create a "ModuleConfig" class at the root of the src folder, that reads like that:

```php
class ModuleConfig
{
    /**
     * @return array
     */
    public function __invoke(): array
    {
        $config = [];
        $config = array_merge_recursive($config, require __DIR__ . '/../config/config.global.php');
        $config = array_merge_recursive($config, require __DIR__ . '/../config/dependencies.global.php');
        return $config;
    }
}
```

This is not a ModuleConfig tied to ZF2 but really an agnostic config that can be used by any container, like Zend Expressive Config.

Can you please update to use this naming and adding the file, for consistency?